### PR TITLE
[Fixit Q4 2022] Bump cdap version and hadoop-common

### PR DESCRIPTION
**Pull request by feefs-google**
Successfully builds, `mvn test` output:
Screen Shot 2022-12-06 at 10 22 56 AM

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -299,8 +299,8 @@
           <version>1.1.0</version>
           <configuration>
             <cdapArtifacts>
-              <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-              <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+              <parent>system:cdap-data-pipeline2_2.11[6.8.0,7.0.0-SNAPSHOT)</parent>
```
Comment (albertshau) [inline pom.xml:302]: this should still be cdap-data-pipeline, not cdap-data-pipeline2_2.11.

It's a bit confusing, but the name here refers to the CDAP artifact name and not a maven artifactId.

Comment (albertshau) [inline pom.xml]: this should still be cdap-data-pipeline, not cdap-data-pipeline2_2.11.

It's a bit confusing, but the name here refers to the CDAP artifact name and not a maven artifactId.

Comment (feefs-google) [reply pom.xml]: Fixed.

**Comment by feefs-google** [comment]
Changed hadoop version to 2.10.2

**Review by albertshau — APPROVED** [review]

**Pull request merged**